packages/backend-common/src/config.ts

```diff
@@ -14,24 +14,32 @@
  * limitations under the License.
  */
 
+import { resolve as resolvePath } from 'path';
+import parseArgs from 'minimist';
+import { Logger } from 'winston';
 import { findPaths } from '@backstage/cli-common';
 import { Config, ConfigReader } from '@backstage/config';
 import { loadConfig } from '@backstage/config-loader';
-import { Logger } from 'winston';
 
 type Options = {
   logger: Logger;
+  // process.argv or any other overrides
+  argv: string[];
 };
 
 /**
  * Load configuration for a Backend
  */
 export async function loadBackendConfig(options: Options): Promise<Config> {
+  const args = parseArgs(options.argv);
+  const configOpts: string[] = args.config ?? [];
```

Comment:
`args.config` is not guaranteed to be an array, right? i see you have a merge function below in the arg parsing, but since this one re-parses it using minimist essentially, is that guarantee maintained here?

Reply:
This parsing is completely separate, but minimist doesn't output an array for single items, so still applies 😁